torch/_torch_docs.py

```diff
@@ -5900,6 +5900,55 @@ def merge_dicts(*dicts):
     tensor([  435.,  1335.,  2235.,  3135.])
 """.format(**multi_dim_common))
 
+add_docstr(torch.nansum,
+           r"""
+nansum(input, dtype=None) -> Tensor
+
+Returns the sum of all elements treating Not a Numbers (NaNs) as zero.
+
+Args:
+    {input}
+
+Keyword args:
+    {dtype}
+
+Example::
+
+    >>> a = torch.randn(1, 3)
+    >>> a
+    tensor([[ 0.1133, -0.9567,  0.2958]])
```

Comment:
Better to construct a tensor like this directly and include a NaN. Maybe something simple like:

```
torch.tensor([1., 2., float('nan'), 4.])
```
?